packages/expo-speech/src/Speech/ExponentSpeech.web.ts

```diff
@@ -100,18 +107,20 @@ export default createWebModule({
       name: voice.name,
       voiceURI: voice.voiceURI,
     }));
-  },
+  }
   async isSpeaking(): Promise<boolean> {
     return window.speechSynthesis.speaking;
-  },
+  }
   async stop(): Promise<void> {
     return window.speechSynthesis.cancel();
-  },
+  }
   async pause(): Promise<void> {
     return window.speechSynthesis.pause();
-  },
+  }
   async resume(): Promise<void> {
     return window.speechSynthesis.resume();
-  },
-  maxSpeechInputLength: MAX_SPEECH_INPUT_LENGTH,
-});
+  }
+  maxSpeechInputLength = MAX_SPEECH_INPUT_LENGTH;
+}
+
+export default registerWebModule('ExpoSpeech', ExpoSpeech);
```

Comment:
Might be nice to have only this 👇. It might be possible with `moduleImplementation.name` – the default would be the class name, but the user can provide a custom name as a static property.
```ts
export default registerWebModule(ExpoSpeech);
```